modules/bridge/src/main/ts/ephox/bridge/api/InlineContent.ts

```diff
@@ -1,27 +1,44 @@
 import {
-  AutocompleterApi,
-  Autocompleter,
-  createAutocompleter,
-  createAutocompleterItem,
-  createSeparatorItem,
-  AutocompleterItemApi,
-  AutocompleterItem,
-  AutocompleterContents,
-  SeparatorItemApi,
-  SeparatorItem,
-  AutocompleterInstanceApi
+  Autocompleter, AutocompleterContents, AutocompleterInstanceApi, AutocompleterItem, AutocompleterItemSpec, AutocompleterSpec, ColumnTypes,
+  createAutocompleter, createAutocompleterItem, createSeparatorItem, SeparatorItem, SeparatorItemSpec
 } from '../components/content/Autocompleter';
+import { ContextPosition, ContextScope } from '../components/content/ContextBar';
```

Comment:
I moved these over from `Toolbar`, as they are inline content UI components similar to the autocompleter. I also splint `ContextToolbar` into `ContextForm` and `ContextToolbar`.